### ISSUE TITLE
Guild-Level Microbiome Signature Associated with COVID-19 Severity and Prognosis

### ISSUE THREAD

**Issue by Omabekee** [waldronlab/BugSigDBcuration#149]
Guild-Level Microbiome Signature Associated with COVID-19 Severity and Prognosis -- Guo M et al. -- mBio

https://journals.asm.org/doi/10.1128/mbio.03519-22

**Comment by zheeburg** [comment]
Isaac
Hi @SvetlanaUP, I'd love to curate this article.

**Comment by zheeburg** [comment]
Thank you very much

**Comment by zheeburg** [comment]
Hi @SvetlanaUP I am done curating this study for review.

**Comment by cmirzayi** [comment]
Hi @zheeburg I think something was wrong here in the curation and I'm not quite sure where things went off track. It looks like you curated the same signatures for every experiment. Is that correct?


| Item | Description | max | points |
|---|---|---|---|
| 1 | All elements marked "Needs review" (none "Incomplete") | 1 | 1 |
| 2 | Correct study design | 1 | 1 |
| 3 | Entered all relevant experiments and no irrelevant experiments | 1 | 0 |
| 4 | Body site correctly identified (i.e. does not include multiple sites) | 1 | 1 |
| 5 | Condition entered according to contrast (correct EFO ontology) | 1 | 1 |
| 6 | Contrast groups correctly identified | 1 | 1 |
| 7 | Groups correctly labeled as 1 and 0 (1=cases, 0=controls) | 1 | 1 |
| 8 | Antibiotic exclusion correctly identified | 1 | 1 |
| 9 | Correctly identified sequencing details | 2 | 0 |
| 10 | Identified correct statistical test | 1 | 0 |
| 11 | Identified MHT correction | 1 | 0 |
| 12 | Correctly recorded matched on factors | 1 | 0 |
| 13 | Entered correct number of statistical tests per experiment | 1 | 0 |
| 14 | All diversity measures identified | 1 | 1 |
| 15 | Diversity results correctly entered as increased/decreased/unchanged | 1 | 1 |
| 16 | All signature sources correctly identified (-1 for each error) | 2 | 0 |
| 17 | Abundance direction correctly selected | 1 | 1 |
| 18 | Members of Signatures identified correctly | 2 | 0 |
| 19 | Correct use of NCBI taxonomy | 2 | 1 |
|  | TOTAL | 23 | 11 |

- Giving you credit since large intestine is close, but the body site should really be feces
- Sequencing was WGS done with Illumina
- Statistical test used in Fig. 2A was KW followed by Dunn's test. No PERMANOVA.
- Dunn's test controls for multiple comparisons
- Figs. 4A, 5B, 6B, 7B entered as signature sources but are incorrect
- Fig. 2A is incorrectly curated. There should be three signatures from the pairwise comparison
- Every signature is just a duplicate of the signature's in experiment 1 it looks like
- No confounders weer controlled for
- Age and sex was matching for the healthy control group

**Comment by AleruDivine** [comment]
Hi @SvetlanaUP, I would like to help out with the correction of this study. Thank you so much!

**Comment by SvetlanaUP** [comment]
@AleruDivine go ahead!

**Comment by fiddyhamma** [comment]
Hello @zheeburg @SvetlanaUP 
Can we work together on correcting this study?

**Comment by AleruDivine** [comment]
Yeah I’m currently working on the corrections @fiddyhamma 😊

**Comment by AleruDivine** [comment]
Good morning @SvetlanaUP, I have finished correcting this study. I opted not to include the matched-on factors since they only pertained to healthy controls. Additionally, I omitted the comparison between COVID-19 and healthy controls due to the absence of alpha diversity measures and differential abundance results.

The comparisons between Mild and Moderate, Mild and Severe, as well as Moderate and Severe, lacked any reference to matched-on factors.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_885 reviewed.

**Comment by AleruDivine** [comment]
Good afternoon @SvetlanaUP 
I've noticed that this study is marked as incomplete. I'd like to review it to correct any mistakes and will report the changes I make, if that's okay with you.

Thank you so much!

**Comment by AleruDivine** [comment]
Thank you @SvetlanaUP 🙏

**Comment by SvetlanaUP** [comment]
@AleruDivine Did you mean Experiment 3 as the second point to delete?

**Comment by AleruDivine** [comment]
Yes, experiment 3. Oh my days!
My apologies @SvetlanaUP .

Thank you so much!

**Comment by cmirzayi** [comment]
Unfortunately I think this might have been a bit of an overcorrection to curate this as a one against all analysis. It's actually a pairwise analysis as denoted by the letters in Figure 2A which is compact letter display (https://en.wikipedia.org/wiki/Compact_letter_display).

There should be 3 experiments.

**Comment by AleruDivine** [comment]
Wow! 🤯 Thank you for sharing this resource, @cmirzayi! I'll update the study accordingly. Does this mean it was correct as it was before, or did I miss something? 🙏

3 experiments were previously curated and reviewed but it got marked as incomplete about a week ago. 
1. Mild vs Moderate
2. Mild vs Severe/Critical
3. Moderate vs Severe/Critical

**Comment by cmirzayi** [comment]
Yeah I flagged it as incorrect as there was something wrong with it then as well so I think you were right to try to fix it but yes there are the 3 experiments you noted.

**Comment by AleruDivine** [comment]
Okay. Thank you so much for clarifying. I’ll make the corrections right away.

**Comment by AleruDivine** [comment]
Hi @cmirzayi @SvetlanaUP 
I have updated the curation, and it is now ready for review
https://bugsigdb.org/Study_885

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_885 reviewed.
